resolver/GlobalPass.cc

```diff
@@ -402,6 +402,9 @@ void Resolver::finalizeSymbols(core::GlobalState &gs) {
         auto sym = core::SymbolRef(&gs, core::SymbolRef::Kind::ClassOrModule, i);
         ENFORCE(sym.isClassOrModule());
         resolveTypeMembers(gs, sym, typeAliases, resolved);
+
+        // Precompute the list of all required ancestors for this symbol
+        sym.data(gs)->computeRequiredAncestorLinearization(gs);
```

Comment:
Should I clear the cache before?

I was expecting this to fail in LSP tests but adding / removing ancestors in `test/testdata/lsp/requires_ancestor.*.rbupdate` seems to work properly, without cash clearing.

Reply:
No, this is where we fill the cache